src/NuGet.Core/NuGet.Common/Errors/NuGetLogCode.cs

```diff
@@ -224,6 +232,46 @@ public enum NuGetLogCode
         NU3502 = 3502,
 
         /// <summary>
+        /// Author cert not valid when timestamped
+        /// </summary>
+        NU3401 = 3401,
+
+        /// <summary>
+        /// Timestamper cert chain does not build
+        /// </summary>
+        NU3402 = 3402,
+
+        /// <summary>
+        /// Timestamper cert does not have the right Enhanced Key Usage
+        /// </summary>
+        NU3403 = 3403,
+
+        /// <summary>
+        /// Timestamp response does not have the right signature value hash
+        /// </summary>
+        NU3404 = 3404,
+
+        /// <summary>
+        /// Timestamp response does not have the right nonce
+        /// </summary>
+        NU3405 = 3405,
+
+        /// <summary>
+        /// Timestamp response does not have a valid hash algorithm oid
+        /// </summary>
+        NU3406 = 3406,
+
+        /// <summary>
+        /// Timestamp response does not have a valid content type
+        /// </summary>
+        NU3407 = 3407,
+
+        /// <summary>
+        /// Timestamp url not passed to sign command
+        /// </summary>
+        NU3901 = 3901,
```

Comment:
Do we have codes for other bad inputs?

In my opinion if there is only 1 error code and no other output, which would be the case for a bad input param, then it should just fail and write that out. 

If this error could show up in a log with 100 other errors, then a code is more critical.